espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/Handlers/BaseHandler.java

```diff
@@ -0,0 +1,24 @@
+package io.appium.espressoserver.lib.Handlers;
+
+import java.util.Map;
+import java.util.UUID;
+
+import fi.iki.elonen.NanoHTTPD.IHTTPSession;
+import io.appium.espressoserver.lib.Http.Response.AppiumResponse;
+import io.appium.espressoserver.lib.Http.Response.BaseResponse;
+import io.appium.espressoserver.lib.Http.Response.InvalidSessionResponse;
+import io.appium.espressoserver.lib.Model.Session;
+
+public abstract class BaseHandler implements RequestHandler {
+    public BaseResponse handle(IHTTPSession session, Map<String, Object> params) {
```

Comment:
https://stackoverflow.com/questions/14694852/can-overridden-methods-differ-in-return-type